crates/forge/Cargo.toml

```
@@ -11,6 +11,7 @@ no_scarb_installed = []
 non_exact_gas_assertions = []
 skip_test_for_only_latest_scarb = []
 skip_test_for_scarb_since_2_11 = []
+run_test_for_scarb_since_2_13 = []
```

Patch version indeed matters in this case, as oracles don't work with 2.13.0:
```suggestion
run_test_for_scarb_since_2_13_1 = []
```